index.js

```diff
@@ -1,23 +1,56 @@
 /**
- * @typedef {import('mdast').Root} Root
- * @typedef {import('retext-smartypants').Options} Options
+ * @typedef {import("mdast").Root} Root
+ * @typedef {import("retext-smartypants").Options} Options
```

Comment:
Can you turn off whatever is doing these changes? I'm not against double quotes in JSDoc, it just doesn't change anything, so it's enlarging the diff. Too bad Prettier can't doesn't enforce this 🤔

Reply:
Seems it was my local prettier extension, reverted